apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -585,11 +462,6 @@ export const FileBrowser = React.memo(() => {
             fileName: newName,
             folderId: file.folderId,
           });
-
-          const [, extension] = newName.split('.');
-          if (!extension) {
-            return codebridgeI18n.noFileExtensionError();
```

Comment:
Did this get handled elsewhere?

Reply:
Oh I meant to comment on this, this code was unreachable after a recent refactor (the line above is a return now). `validateFileName` handles checking the extension